html-forms/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Links</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <section>
+        <div>
+        <label for="email">Email</label>
+        <p><input id="email" type="email" name="email"></p>
+      </div>
+      </section>
+
+      <section>
+        <p>
+        <label for="select"></label>
+          <span>Why is coding cool?</span>
+      </p>
```

Comment:
You can omit the span tag and put it's text content inside the label like this:

`<label for="select">Why is coding cool?</label>`